MIP/mip-18/README.md

```diff
@@ -0,0 +1,99 @@
+
+### Multisig Security Considerations
+
+- Believe it is best practice to rely on a service that is fully dedicated to multisigs. Eventual exploits found might open us up to exploits while safe.global could've prevent the exploit for us.
+- Having full control of our multisig deployments is dangerous and more engineering debt.
+- Having manual verification might stop us from automating a exploit that we did not notice.
+- Both multisigs serve as a dual signing authorities, ensuring a strict separation between proposing and executing transactions.
+- The timelock ensures a sufficient review window for all parties, with the ability for either multisig to cancel the transaction during the delay period.
+- Multisig can fully transition to a fully automated KMS setup once we have that in place. MOVE Token deployment did not follow all procedures, dependency contracts were deployed by an EOA.
+- Movement Foundation Safe will have ownership of Token Upgradeability - as far as I understood, please correct me if I'm wrong.
+
+## Reference Implementation
+
+The multisigs will be deployed using the Safe global multisig service, with contracts interacting through the SafeKit API. The remote key will automate transaction proposals for review by Core Contributors engineers.
+
+## Verification
+
+1. **Correctness**:  
+   This MIP follows the Safe multisig architecture, which has been extensively vetted in decentralized governance.
+
+2. **Security Implications**:  
+   - A remote key introduces automation but requires careful monitoring to prevent unauthorized transactions.
+   - The separation of proposal and execution roles reduces the risk of unilateral decision-making.
+   - If any of the keys part of each multisig are compromised, we must quickly remove it and substitute it with another key.
+
+3. **Performance Impacts**:  
+   - The additional timelock delay adds a slight performance lag but is necessary to enhance the security and transparency of upgrades.
+   - Upgrades should not be periodic. This means that manual procedures might be more interesting than full automation.
+
+4. **Validation Procedures**:  
+   - The Safe multisig service and timelock mechanisms have been validated by various communities in the Ethereum ecosystem.
+   - It's not possible to fully automate this service because Movement Founadtion will always require the final signatures.
+
+5. **Peer Review and Community Feedback**:  
+   Current Governance proposals have indicated that we should have full automation and no access to signing keys. Currently we are a bit far from achieving full automation.
+
+## Errata
+
+Any post-publication corrections or updates to this MIP will be documented here to maintain transparency and accuracy.
+
+## Appendix
+
+- [R1] Safe Multisig: https://safe.global/
+- [R2] SafeKit API Documentation: https://docs.safe.global/sdk/api-kit
```

Comment:
Thanks for the flow chart. In the case that a proposal is not signed on by at least 4 / 5 engineers, a proposer can just re-submit that proposal again? I'm just thinking in the case that an engineer missed the timelock period for some reason